internal/controllers/addon/utils.go

```diff
@@ -140,6 +140,14 @@ func reportLastObservedAvailableCSV(addon *addonsv1alpha1.Addon, csvName string)
 	addon.Status.LastObservedAvailableCSV = csvName
 }
 
+func reportAddonInstalledCondition(addon *addonsv1alpha1.Addon, conditionStatus metav1.ConditionStatus) {
+	addon.Status.Conditions = []metav1.Condition{
+		{
+			Status: conditionStatus,
```

Comment:
Please add Type, Reason, Message and Observed generation, like here:
https://github.com/openshift/addon-operator/blob/a6b8c51f231d0144702e8b695191b5ce985e6265/internal/controllers/addon/utils.go#L230-L234

Ref Kube API Conventions:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties